app/jobs/v3/create_service_instance_job.rb

```diff
@@ -23,19 +33,129 @@ def operation_type
         'create'
       end
 
-      def send_broker_request(client)
-        client.provision(
-          service_instance,
-          accepts_incomplete: true,
-          arbitrary_parameters: arbitrary_parameters,
-          maintenance_info: service_instance.service_plan.maintenance_info,
-          user_guid: user_audit_info.user_guid
+      def max_attempts
+        1
+      end
+
+      def display_name
+        'service_instance.create'
+      end
+
+      def resource_type
+        'service_instances'
+      end
+
+      def resource_guid
+        @service_instance_guid
+      end
+
+      def perform
+        not_found! unless service_instance
+
+        raise_if_other_operations_in_progress!
+
+        compute_maximum_duration
+
+        begin
+          if @first_time
+            @first_time = false
+            action.provision(service_instance, parameters: @arbitrary_parameters, accepts_incomplete: true)
+            compatibility_checks # TODO: get the warning from the action?
```

Comment:
What's the plan for this TODO?

Reply:
Removed it in the benefit of completing this work.